src/ProgressOnderwijsUtils/MetaObject/IPropertiesAreUsedImplicitly.cs

```diff
@@ -0,0 +1,17 @@
+﻿using JetBrains.Annotations;
+
+namespace ProgressOnderwijsUtils
+{
+    [UsedImplicitly(ImplicitUseTargetFlags.Members)]
+    public interface IPropertiesAreUsedImplicitly
+    {
+        /*
+         * This doesn't really work well, since resharper thinks that objects with implicitly
+         * used members are used themselves.
+         * 
+         * My suggestion is to remove the tags at the top of this interface, and instead demand
+         * (by test) that all the properties of inheritors have the attribute used implicitly
+         * (or similar)
+         */

```

Comment:
even aside: heeft `IMetaObject` nog bestaansrecht hierna?

Reply:
Ja, want die gebruiken we voor lezen. Misschien kan de naam wel beter

Reply:
Maar er hangt geen `UsedImplicitly` meer aan?  dus het is alleen nog "ter indicatie"?

Reply:
Mag ook weg wmb